search-app/docs/api.md

```diff
@@ -428,7 +434,7 @@ For private collection, an EDL bearer token or a Launchpad token can be used to
 %CMR-ENDPOINT%/concepts/<concept-id>?token=EDL-xxxxxx
 ```
 
-#### <a name="atom"></a> Atom
+#### <a name="atom"></a> Atom XML
```

Comment:
we seam to be talking about the differences between our formats, so I am adding text to make things slightly more clear